chart/templates/deployment.yaml

```diff
@@ -108,11 +108,11 @@ spec:
         - name: CERT_PATH
           value: {{ .Values.activeMQ.cert_mount_path | quote }}
         - name: KEY_FILE_NAME
-          value: client_key
+          value: {{ .Values.activeMQ.client_key | quote }}
```

Comment:
This will expose secret value in the deployment, current `client_key` is defined in the secret msg-certs, please check 

https://github.com/RedHatQE/notify-service/blob/main/chart/templates/deployment.yaml#L179
https://github.com/RedHatQE/notify-service/blob/main/chart/templates/deployment.yaml#L204